tests/hash-backend-integration/src/tests/graph/knowledge/primitive/entity.test.ts

```diff
@@ -222,34 +238,40 @@ describe("Entity CRU", () => {
     );
   });
 
-  it("can read all latest entities", async () => {
+  it("can read all latest person entities", async () => {
     const allEntities = await graphApi
       .getEntitiesByQuery(testUser.accountId, {
         filter: {
-          all: [],
+          all: [
+            {
+              equal: [
+                { path: ["ownedById"] },
+                { parameter: testOrg.accountGroupId },
+              ],
+            },
+            {
+              endsWith: [
+                { path: ["type(inheritanceDepth=0)", "baseUrl"] },
+                {
+                  parameter: `/types/entity-type/person/`,
```

Comment:
I haven't seen this shorthand before... does it (1) add the frontend URL or (2) match any type base URL which ends with this string? e.g. would `https://blockprotocol.org/types/entity-type/person/` be matched?

Reply:
It's treating `baseUrl` as a string, so your example would be matched.

Reply:
That also works with`versionedUrl` btw, so you could ask for `"/types/entity-type/person/v/1"`.

Reply:
Understood, thank you